packages/expo-image/src/ExpoImage.web.tsx

```diff
@@ -10,7 +10,7 @@ import useSourceSelection from './web/useSourceSelection';
 loadStyle();
 
 export const ExpoImageModule = {
-  async prefetch(urls: string | string[], _): Promise<boolean> {
+  async prefetch(urls: string | string[], _, __): Promise<boolean> {
```

Comment:
```suggestion
  async prefetch(urls: string | string[], _): Promise<boolean> {
```

Reply:
`ExpoImageModule` is for web instead of the native module, I know.

I didn't change the 3rd parameter of 'prefetch()' function in the native module.

so I don't think it's necessary to remove the 3rd parameter of web's function.

Reply:
> You added headers as a separate argument in the previous version. Currently, the function no longer receives three arguments, so the web implementation can be adjusted accordingly 😅

I don't understand...🤔
https://github.com/toy0605/expo/blob/d94d6987a8f364f61fdfbac372b3d1a2360248e6/packages/expo-image/src/Image.tsx#L63

```ts
return ExpoImageModule.prefetch(Array.isArray(urls) ? urls : [urls], cachePolicy, headers);
```

I called the `prefetch()` function with a third parameter(HTTP header).
I didn't change the parameters of function in the native module.
I changed **the `prefetch()` function of `Image.tsx`**, not native's functions.

As far as I know, I think the `prefetch()` function for both native and web receives three argument.
This is my code flows if I wasn't wrong.
```mermaid
flowchart LR
   A["Image.prefetch(urls, options)\n(2 arguments)\nfrom Image.tsx"]
   B["Call ExpoImageModule.prefetch(Array.isArray(urls) ? urls : [urls], cachePolicy, headers)\n(Call with 3 arguments)\nfrom Image.tsx"]
   C["ExpoImageModule.prefetch(urls: string | string[], _, __)\n(If it should be explicit, 3 arguments.\nIf unused arguments should be removed, 1 argument.)\nfrom ExpoImage.web.tsx"]
   D["ImageModule.prefetch(urls: [URL], cachePolicy: ImageCachePolicy, headersMap: [String, String]?, promise: Promise)\n(3 arguments without Promise)\nfrom ImageModule.swift"]
   E["ExpoImageModule.prefetch(urls: List<string>, cachePolicy: CachePolicy, headersMap: Map<string, string>?, promise: Promise)\n(3 arguments without Promise)\nfrom ExpoImageModule.kt"]
   A-->B
   B-->C
   B-->D
   B-->E
```

If you want the `prefetch()` function parameters to be without unused parameters, I also removed the second parameter. (I'm not sure why there was a second parameter before either...)

...or was I wrong?🤔
If I was wrong, I have to change the `prefetch()` function in `Image.tsx` or the `prefetch()` function in the native modules.

Reply:
Sorry, I thought you combined options and headers everywhere. My suggestion was wrong.